src/main/java/mServer/crawler/sender/arte/MediathekArte_de.java

```diff
@@ -55,6 +55,7 @@
     protected String URL_CONCERT_NOT_CONTAIN = "-STF";
     protected String TIME_1 = "<li>Sendetermine:</li>";
     protected String TIME_2 = "um";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this field "TIME_2" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00116)